about.html

```diff
@@ -12,39 +12,39 @@
 <body>
     <header class="header">
         <div class="miniNav">
-            <nav>
+            <nav class="miniItems">
                 <a href="#"><i class="fa-brands fa-facebook-f"></i></a>
                 <a href="#"><i class="fa-brands fa-twitter"></i></a>
-                <a href="">English</a>
-                <a href="">My Page</a>
-                <a href="">Log out</a>
+                <a href="#">English</a>
+                <a href="#">My Page</a>
+                <a href="#">Log out</a>
             </nav>
         </div>
         <div class="menuButton">
             <div class="menuBar"></div>
         </div>
-        <div class="logo"><a href="#"><img src="images/Techscape_logo.png" alt="techscape logo" width="40px" height="40px" /></a></div>
         <nav class="menuItems">
+            <a id='logo' href="index.html"><div class="logo"></div></a>
             <a href="about.html">About</a>
-            <a href="index.html">Program</a>
-            <a href="">Join</a>
-            <a href="">Sponsor</a>
-            <a href="">Newsletter</a>
-        </nav>
-        <button id='ccCampaign' type="submit">CC Campaign</button> 
+            <a href="#">Program</a>
+            <a href="#">Join</a>
+            <a href="#">Sponsor</a>
+            <a href="#">Newsletter</a>
+            <button id='ccCampaign' type="submit">CC Campaign</button>
+        </nav> 
     </header>
     <section id="about">
         <h1>
             <small id="aboutSmall">"Hello!Sharing World"</small><br>
             TECHSCAPE@3AM<br>GLOBAL SUMMIT 2023
         </h1>
+        <span class="redBar"></span>
         <div class="aboutUs">
             <p>The Global Summit brings together the community of experts, academics, and activists who comprise the Techscape@3AM affiliate network in a different country every two years. This year, we hope to expand our invitation list including organizations and individuals</p>
         </div>
         <p class="contact">
-            Please contact us per email for any further questions<br>
-            about golobal submit 2023!
-            <address><a href="techscape99@gmail.com">techscape99@gmail.com</a></address>
+            Please contact us per email for any further questions about golobal submit 2023!
+            <address><a id="address" href="techscape99@gmail.com">techscape99@gmail.com</a></address>
         </p>
     </section>
```

Comment:
- [ ] On the mobile version of the About page first section, there is an overlap happening between the first and second section texts. This happens on some mobile screens and it is probably due to a `height` property with a `vh` value. Kindly fix the styling of this section so there is no overlap in any screen size. Be aware of the use of `vh` or `vw` values when sizing elements, screen sizes can be different and it is hard to make it responsive for all screen sizes.
 - Your design:
 - 
<img width="482" alt="image" src="https://user-images.githubusercontent.com/61919617/225436591-0e2d5824-f38f-4f04-8fd9-77f68f101772.png">

 - Template design:
 - 
<img width="395" alt="image" src="https://user-images.githubusercontent.com/61919617/225436689-7c795e5d-10d8-4b16-8f77-164f3c91f6b6.png">

Reply:
## :+1: